cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go

```diff
@@ -712,7 +712,9 @@ func genComment(out io.Writer, lines []string, indent string) {
 		lines = lines[:l-1]
 	}
 	for _, c := range lines {
-		fmt.Fprintf(out, "%s// %s\n", indent, c)
+		if c != "+optional" {
```

Comment:
This will tend to be a lot of special cases - we may just want to avoid this for now.  I think it's ok that the protobuf output has this.